File: Chapter_3/project_3_1.c
#include <stdio.h>

int main(void) {
  int day, month, year;
  printf("Enter a date in dd/mm/yyy format: ");
  scanf("%d/%d/%d", &day, &month, &year);

  printf("%d/%d/%d\n", year, month, day);
}

File: Chapter_3/project_3_1.pl
use strict;
use warnings;
use feature 'say';

use Time::Piece;

print "Enter a date in dd/mm/yyy format: ";
chomp( my $user_date = <STDIN> );

my $tp = Time::Piece->strptime( $user_date, '%d/%m/%Y' );

say $tp->strftime('%Y/%m/%d');
